voyager_archive/views_utils.py

```diff
@@ -8,22 +8,56 @@ def get_auth_record(auth_id: int) -> list:
 
     return get_marc_fields(marc_text)
 
+
 def get_bib_record(bib_id: int) -> list:
     marc_text = get_object_or_404(BibRecord, bib_id=bib_id).bib_record
 
     return get_marc_fields(marc_text)
 
+
 def get_mfhd_record(mfhd_id: int) -> list:
     marc_text = get_object_or_404(MfhdRecord, mfhd_id=mfhd_id).mfhd_record
-   
+
     return get_marc_fields(marc_text)
 
+
 def get_marc_fields(marc_text: str) -> list:
-    byte_obj = marc_text.encode('utf-8')
+    byte_obj = marc_text.encode("utf-8")
     marc_record = Record(data=byte_obj)
     marc_fields = marc_record.get_fields()
 
-    return marc_fields
+    marc_field_list = []
+    marc_record_dict = {}
+    marc_record_dict["leader"] = marc_record.leader
+
+    for field in marc_fields:
+
+        tmp_rec_dict = {}
+        tmp_rec_dict["tag"] = field.tag
+        if hasattr(field, "data"):
+            tmp_rec_dict["data"] = field.data
+
+        if hasattr(field, "indicator1"):
+            tmp_rec_dict["indicator1"] = str.strip(field.indicator1)
```

Comment:
Instead of removing blank indicators, then displaying underscores in the template, do that replacement here:
```
tmp_rec_dict["indicator1"] = field.indicator1.replace(" ", "_")
tmp_rec_dict["indicator2"] = field.indicator2.replace(" ", "_")
```